source/Playnite/SelectableItem.cs

```diff
@@ -490,6 +501,36 @@ public class SelectableDbItemList : SelectableIdItemList<DatabaseObject>, INotif
     {
         private readonly bool includeNoneItem;
 
+        private bool searchCheckedState = false;
+        public bool SearchCheckedState

```

Comment:
I think it would be better to rename this to something more clear. Like `ShowSelectedOnly` or something like that.